README.rst

```diff
@@ -548,6 +548,10 @@ Choose from the list of available rules:
   - ``keep_multiple_spaces_after_comma`` (``bool``): whether keep multiple spaces
     after comma; defaults to ``false``
 
+* **method_chaining_indentation**
+
+  Method chaining MUST be properly indented.
```

Comment:
could you add this rule to `.php_cs.dist` file to show that it works nicely and do not change properly intended code ?